components/sharing-dialog/src/__tests__/ShareBlock.test.js

```diff
@@ -13,7 +14,9 @@ describe('SharingDialog widget - ShareBlock component', () => {
 
     const getShareBlockComponent = props => {
         if (!shallowShareBlockComponent) {
-            shallowShareBlockComponent = shallow(<ShareBlock {...props} />)
+            shallowShareBlockComponent = mount(<ShareBlock {...props} />, {
+                wrappingComponent: CustomDataProvider,
+            })
```

Comment:
It seems to me that this should also work with shallow. It also has the wrappingComponent API. But I'm a react-query missing provider error, so I went with mount. Costs a little more time to run.